src/backend/cod1.c

```diff
@@ -4482,7 +4482,16 @@ code *loaddata(elem *e,regm_t *pretregs)
 #endif
         assert(forregs & BYTEREGS);
         if (!I16)
+        {
+            unsigned fl = e->EV.sp.Vsym->Sfl;
+            if (config.target_cpu >= TARGET_PentiumPro && config.flags4 & CFG4speed &&
+                !(config.exe & EX_OSX64)
```

Comment:
What about the OSX part here? Is that b/c of `mov GOT` linker rewrites? In any case it needs a comment and probably applies to other PIC code as well.